libraries/SRV_Channel/SRV_Channel_aux.cpp

```diff
@@ -46,7 +46,13 @@ void SRV_Channel::output_ch(void)
             if (SRV_Channels::passthrough_disabled()) {
                 output_pwm = c->get_radio_trim();
             } else {
-                output_pwm = c->get_radio_in();
+                // if the input value has not been changed, do nothing
+                uint16_t radio_in = c->get_radio_in();
+                if (previous_pwm[passthrough_from] < radio_in - c->get_dead_zone() ||
```

Comment:
I think this is vulnerable to integer underflow, eg. if radio_in is zero

Reply:
thank you.
I added the condition "radio_in >= c->get_dead_zone()"